materialize-snowflake/snowflake.go

```diff
@@ -875,6 +869,43 @@ func (d *transactor) deleteFiles(ctx context.Context, files []string) error {
 	return nil
 }
 
+func (d *transactor) cleanupPipes(ctx context.Context, exclude []string) error {
+	var query = "SELECT PIPE_CATALOG, PIPE_SCHEMA, PIPE_NAME FROM INFORMATION_SCHEMA.PIPES WHERE PIPE_NAME LIKE 'FLOW_PIPE_%';"
```

Comment:
Two related questions:
- Is this going to remove pipes used by other materializations going to the same Snowflake database?
- How will this interact with split task shards for the same materialization?

Reply:
@williamhbaker both valid points... this would delete pipes from other materializations and also from split shards. I'm going to change this to only look for pipes with the same binding, keyBegin and table name, but with a different version 🤔